assets/css/02-baseline.css

```diff
@@ -9,6 +9,7 @@
   ::after,
   ::before,
   html {
+    text-align: center;
```

Comment:
This probably should not be here as some of the pages are left aligned (insights and forms)